aws/registers/register_school_authority.tf

```diff
@@ -0,0 +1,58 @@
+module "school-authority_policy" {
+  source = "../modules/instance_policy"
+  id = "school-authority"
+  enabled = "${signum(lookup(var.instance_count, "school-authority"))}"
+
+  vpc_name = "${var.vpc_name}"
+  vpc_id = "${module.core.vpc_id}"
+}
+
+module "school-authority_presentation" {
```

Comment:
This needs to be changed to fit the new (openregister) style. Me or @michaelabenyohai can help you with this.